packages/adaptive_scaffold/example/lib/main.dart

```diff
@@ -521,25 +611,215 @@ class _DetailTile extends StatelessWidget {
+                                    ),
+                                    const SizedBox(width: 15),
+                                    Container(
+                                        padding: const EdgeInsets.all(8.0),
+                                        decoration: const BoxDecoration(
+                                            color: Colors.white,
+                                            borderRadius: BorderRadius.all(
+                                                Radius.circular(15))),
+                                        child: Icon(Icons.more_vert,
+                                            color: Colors.grey[600]))
+                                  ],
+                                )),
+                          ]),
+                      const SizedBox(height: 20),
+                    ],
+                  )),
+              Expanded(
+                  child: ListView.builder(
+                      itemCount: item.emails!.length,
+                      itemBuilder: (BuildContext context, int index) {
+                        final _Email thisEmail = item.emails![index];
+                        return _EmailTile(
+                            sender: thisEmail.sender,
+                            time: thisEmail.time,
+                            senderIcon: thisEmail.image,
+                            recepients: thisEmail.recepients,
+                            body: thisEmail.body,
+                            bodyImage: thisEmail.bodyImage);
+                      })),
+              //Text(item.body, style: Theme.of(context).textTheme.bodyLarge),
+            ],
+          ),
+        ),
+      ),
+    );
+  }
+}
+
+class _EmailTile extends StatelessWidget {
+  const _EmailTile({
+    required this.sender,
+    required this.time,
+    required this.senderIcon,
+    required this.recepients,
+    required this.body,
+    required this.bodyImage,
+    Key? key,
+  }) : super(key: key);
+
+  final String sender;
+  final String time;
+  final String senderIcon;
+  final String recepients;
+  final String body;
+  final String bodyImage;
+
+  @override
+  Widget build(BuildContext context) {
+    return Padding(
+      padding: const EdgeInsets.fromLTRB(0, 4, 0, 4),
+      child: Container(
+        decoration: const BoxDecoration(
+            color: Colors.white,
+            borderRadius: BorderRadius.all(Radius.circular(10))),
+        child: Padding(
+          padding: const EdgeInsets.all(16),
+          child: Column(
+            crossAxisAlignment: CrossAxisAlignment.start,
+            children: <Widget>[
+              Row(
+                children: <Widget>[
+                  CircleAvatar(
+                      radius: 18,
+                      child: Image.asset(
+                        senderIcon,
+                        width: 100,
+                        height: 100,
+                        fit: BoxFit.cover,
+                      )),
```

Comment:
Nit: `)),`-> `),),` I will stop pointing these out but try to do a run through the main.dart and if you find similar issues with indentation and issues that look like this try to add commas and format.